src/app/(auth)/chats/conversation/[id].tsx

```diff
@@ -45,20 +42,36 @@ export default function Page() {
 
   const sendMutation = useMutation({
     mutationFn: async (message) => {
-      const res = await sendChatMessage(id, message[0].text)
-      const msg = {
-        _id: res.id,
-        id: res.id,
-        createdAt: new Date(),
-        sent: true,
-        text: message[0].text,
-        user: {
-          _id: selfUser.id,
-          name: selfUser.username,
-          avatar: selfUser.avatar,
-        },
+      console.log('send message', { message, id })
```

Comment:
intentionally left in?

Reply:
at first yet, but the code in including this line worked - the issue #234 is in the request that comes after. So I'll delete this log line because it served its purpose.